.changes/shell-execute-extra-newline.md

```diff
@@ -0,0 +1,6 @@
+---
+"@tauri-apps/api": "patch:bug"
```

Comment:
does this also need to be minor?

Problem would be then that we have 1.6 in js and 1.7 in rust. Maybe we should copy the last 1.5 release into 1.6 to fix that.

Reply:
The problematic change was in the other change file where it is bumped as minor.

Reply:
Ah okay, that leaves the version mismatch question though. Maybe i should move that to discord though as it's not directly related to this PR.